packages/contracts-bedrock/test/DeployOPChain.t.sol

```diff
@@ -399,6 +410,15 @@ contract DeployOPChain_Test is DeployOPChain_TestBase {
         basefeeScalar = uint32(uint256(hash(_seed, 6)));
         blobBaseFeeScalar = uint32(uint256(hash(_seed, 7)));
         l2ChainId = uint256(uint256(hash(_seed, 8)));
+        uint256 numStartingAnchorRoots = bound(uint256(uint256(hash(_seed, 9))), 1, 10);
```

Comment:
Can we add a comment with rationale for these bounds? e.g. why not a lower bound of 0, and what's the max upper bound we might see in practice?

Reply:
Change this to add one root per game type per expected usage.